common/tools/dev-tool/src/util/testProxyUtils.ts

```diff
@@ -87,37 +205,31 @@ export async function isProxyToolActive(): Promise<boolean> {
   }
 }
 
-async function makeRequest(uri: string, requestOptions: RequestOptions): Promise<IncomingMessage> {
-  return new Promise<IncomingMessage>((resolve, reject) => {
-    const req = request(uri, requestOptions, resolve);
-    req.once("error", reject);
-    req.end();
-  });
-}
+async function getTargetVersion() {
+  return "1.0.0-dev.20230126.1";
 
-async function getImageTag() {
   // Grab the tag from the `/eng/common/testproxy/target_version.txt` file [..is used to control the default version]
   // Example content:
   //
   // 1.0.0-dev.20220224.2
   // (Bot regularly updates the tag in the file above.)
-  try {
-    const contentInVersionFile = await fs.readFile(
-      `${path.join(await resolveRoot(), "eng/common/testproxy/target_version.txt")}`,
-      "utf-8"
-    );
-
-    const tag = contentInVersionFile.trim();
-    if (tag === undefined) {
-      throw new Error();
-    }
-
-    log.info(`Image tag obtained from the powershell script => ${tag}\n`);
-    return tag;
-  } catch (_: any) {
-    log.warn(
-      `Unable to get the image tag from the powershell script, trying "latest" tag instead\n`
-    );
-    return "latest";
-  }
+  // try {
```

Comment:
revert this too?

Reply:
The target_version.txt value hasn't been updated yet, but there's a PR for it. I'll uncomment this and merge as soon as it's ready to go.